src/pages/iou/ReceiptDropUI.tsx

```diff
@@ -9,25 +8,20 @@ import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
 import CONST from '@src/CONST';
 
-const propTypes = {
-    /** Callback to execute when a file is dropped. */
-    onDrop: PropTypes.func.isRequired,
+type ReceiptDropUIProps = {
+    /** Function to execute when an item is dropped in the drop zone. */
+    onDrop: (event: DragEvent) => void;
 
-    /** Pixels the receipt image should be shifted down to match the non-drag view UI */
-    receiptImageTopPosition: PropTypes.number,
+    receiptImageTopPosition?: number;
 };
 
-const defaultProps = {
-    receiptImageTopPosition: 0,
-};
-
-function ReceiptDropUI({onDrop, receiptImageTopPosition}) {
+function ReceiptDropUI({onDrop, receiptImageTopPosition}: ReceiptDropUIProps) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
     return (
         <DragAndDropConsumer onDrop={onDrop}>
             <View style={[styles.receiptDropOverlay, styles.w100, styles.h100, styles.justifyContentCenter, styles.alignItemsCenter]}>
-                <View style={styles.receiptImageWrapper(receiptImageTopPosition)}>
+                <View style={styles.receiptImageWrapper(receiptImageTopPosition ?? 0)}>
```

Comment:
Should we define default prop rather than null coalescing at occurrences of `receiptImageTopPosition`?